dashboard/test/controllers/api/v1/pd/application/principal_approval_applications_controller_test.rb

```diff
@@ -16,9 +16,9 @@ class PrincipalApprovalApplicationsControllerTest < ::ActionController::TestCase
     end
 
     PRINCIPAL_APPROVAL_EMAILS = [
-      :principal_approval_completed,
```

Comment:
What are your thoughts on changing `PRINCIPAL_APPROVAL_EMAILS` to `ADMIN_APPROVAL_EMAILS`? It could be clearer since all its contents are now "admin..." but also there's tons of other variables we are not changing that say principal even in this file so maybe it's best to leave that alone? Either way is totally good and it's super minor, just was curious on your thoughts since it only relates to the emails and I believe only shows up in this file.